LayoutTests/inspector/debugger/async-stack-trace-basic.html

```diff
@@ -65,6 +79,16 @@
 {
     let suite = InspectorTest.createAsyncSuite("AsyncStackTrace.Basic");
 
+    addAsyncStackTraceTestCase(suite, {
+        name: "AsyncStackTrace.Basic.Microtask",
+        expression: "triggerMicrotask()",
```

Comment:
Totally unimportant, but I always tried to write JS expressions / code snippets with template strings. I thought that distinction was both useful (in case the code had double quoted strings in it) and nice (you could distinguish just a regular property from code being run in the inspected page).

```suggestion
        expression: `triggerMicrotask()`,
```

Feel free to ignore =)

Reply:
oh lol oops this was a bad copypasta :P

i try to do the same thing so ill change :)